.github/workflows/delete-model-runs.yaml

```diff
@@ -0,0 +1,58 @@
+# Workflow that can be manually dispatched to delete test model runs that
+# do not need to be persisted indefinitely.
+#
+# Gated such that it's impossible to delete runs older than the current
+# assessment cycle, where each assessment cycle starts in April.
+
+name: delete-model-runs
+
+on:
+  workflow_dispatch:
+    inputs:
+      run-ids:
+        description: >
+          Comma-delimited list of IDs of model runs to delete (no spaces). Note
+          that the workflow assumes these IDs correspond to model runs for the
+          current upcoming assessment cycle, and if that's not the case the
+          deletion script will raise an error.
+        required: true
+        type: string
+
+jobs:
+  delete-model-runs:
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to interact with GitHub's OIDC Token endpoint so we can auth AWS
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout repo code
+        uses: actions/checkout@v4
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install system dependencies
+        run: sudo apt-get install libgit2-dev
+        shell: bash
+
+      - name: Disable renv sandbox to speed up install time
+        run: echo "RENV_CONFIG_SANDBOX_ENABLED=FALSE" >> .Renviron
+        shell: bash
```

Comment:
I don't fully understand the implications of disabling the renv sandbox, but according to my read of [the docs](https://rstudio.github.io/renv/reference/sandbox.html) it seems to be unnecessary in an isolated environment like GitHub Actions, and disabling it reduces renv install time from [1m7s](https://github.com/ccao-data/model-res-avm/actions/runs/6897471429/job/18765700965#step:5:1) to [7s](https://github.com/ccao-data/model-res-avm/actions/runs/6908697011/job/18798517688#step:6:1).